src/LineaStateBridge.sol

```diff
@@ -136,30 +137,17 @@ contract LineaStateBridge is Ownable2Step {
         emit RootPropagated(latestRoot);
     }
 
-    /// @notice Adds functionality to the StateBridge to transfer ownership
-    /// of LineaWorldID to another contract on L1 or to a local Linea Stack chain EOA
-    /// @dev  _messageService should be hardcoded to 0x508Ca82Df566dCD1B0DE8296e70a96332cD644ec for L2 Linea
-    /// @dev _remoteSender should be address of L1 LineaStateBridge
-    /// @custom:revert if _owner is set to the zero address
-    function transferOwnershipLinea(address _messageService, address _remoteSender) external onlyOwner {
-        if (_remoteSender == address(0) || _messageService == address(0)) {
-            revert AddressZero();
-        }
-
-        // The `encodeCall` function is strongly typed, so this checks that we are passing the
-        // correct data to the Linea Stack chain bridge.
-        bytes memory message =
-            abi.encodeCall(MessageServiceBase.updateMessageServiceBase, (_messageService, _remoteSender));
-
-        IMessageService(messageServiceAddress).sendMessage(
-            // Contract address on the Linea Stack Chain
-            lineaWorldIDAddress,
-            _gasLimitTransferOwnership,
-            message
-        );
+    /// @notice Allows for ownership to be transferred with specifying the locality.
+    /// @param _owner   The new owner of the contract.
+    /// @param _isLocal Configures the locality of the ownership.
+    function transferOwnership(address _owner, bool _isLocal) external onlyOwner {
+        LineaWorldID(lineaWorldIDAddress).transferOwnership(_owner, _isLocal);
```

Comment:
Unfortunately I don't think it will work this way, because LineaWorldID lives on another chain, so we still need MessageService. We just need to encode calls to the ICrossDomainOwnableLinea (or ILineaWorldID, depending on the method) instead of IMessageServiceBase